### DISCUSSION TITLE
Differences between spring animations on 60 fps and 120fps screens.

### DISCUSSION THREAD

**Discussion by xzilja** [software-mansion/react-native-reanimated#4897] [Q&A]
I came across a weird bug today (is it even a bug?) where I was testing animations on simulator and got that slight spring bounce I was looking for. When deployed on my phone, animation happened, but I lost the "bounce" effect. After fiddling around I came to conclusion that it was due to my device having 120fps screen. When I switched to battery saving mode and screen went to 60fps it worked same as in simulator.

So I wanted to ask if fps can affect spring animation behavior and if so, are there ways around it so that they look consistent on devices of various fps ranges?

**Comment by piaskowyk** [comment]
Animations are based on current timestamp. They should work in exactly the same no matter how FPS has your screen
